src/Microsoft.SqlTools.ServiceLayer/LanguageServices/Contracts/Completion.cs

```diff
@@ -58,7 +58,7 @@ public class Command
         /// <summary>
         /// The identifier of the actual command handler, like `vsintellicode.completionItemSelected`.
         /// </summary>
-        public string CommandStr { get; set; }
+        public string command { get; set; }
```

Comment:
Let's also add Title and Tooltip as well to fully complete the vscode spec.

Reply:
Where are Title/Tooltip defined in the spec?

Reply:
This is for the Command interface - which is what you're modifying here. 

https://github.com/microsoft/vscode/blob/master/src/vs/vscode.d.ts#L19

Reply:
Added Title/Tooltip in the Command interface.